azkaban-common/src/main/java/azkaban/metrics/CommonMetrics.java

```diff
@@ -34,6 +34,8 @@
   private final MetricsManager metricsManager;
   private Meter dbConnectionMeter;
   private Meter flowFailMeter;
+  private Meter dispatchFailMeter;
```

Comment:
Sounds a not bad idea, but it is difficult to do at today's codebase, since Executor still exists in common module.